src/test/java/com/google/edith/LoginServletTest.java

```diff
@@ -41,75 +33,84 @@
 import org.mockito.MockitoAnnotations;
 
 public final class LoginServletTest {
-  private Map<String, Object> map =
-      ImmutableMap.of("com.google.appengine.api.users.UserService.user_id_key", "12345");
 
-  private final LocalServiceTestHelper loggedInTestHelper =
-      new LocalServiceTestHelper(new LocalUserServiceTestConfig())
-          .setEnvAttributes(map)
-          .setEnvIsLoggedIn(true)
-          .setEnvAuthDomain("gmail")
-          .setEnvIsAdmin(true)
-          .setEnvEmail("user@gmail.com");
-
-  private final LocalServiceTestHelper loggedOutTestHelper =
-      new LocalServiceTestHelper(new LocalUserServiceTestConfig()).setEnvIsLoggedIn(false);
-
-  private final UserService userService = UserServiceFactory.getUserService();
   private LoginServlet loginServlet;
 
   @Mock HttpServletRequest request;
 
   @Mock HttpServletResponse response;
 
+  @Mock LoginInterface loginImpl;
```

Comment:
nit: loginService or similar to avoid abbreviations

Reply:
Changed it to ```loginImplementation```